tests/unit/aiplatform/test_pipeline_jobs.py

```diff
@@ -29,6 +29,10 @@
 from google.cloud import aiplatform
```

Comment:
Please also update the experiment system tests to test this functinality. Add a test right after this test: https://github.com/googleapis/python-aiplatform/blob/main/tests/system/aiplatform/test_experiments.py#L263

It should be able to get the `pipeline_job` using `job = aiplatform.PipelineJob(self._pipeline_job_id` and then verify the associated experiment is `self._experiment_name`.

Reply:
I added a call to `get_associated_experiment` to `test_add_pipeline_job_to_experiment`. Just kicked off our system tests from my branch.